src/main/java/org/broadinstitute/consent/http/service/InstitutionService.java

```diff
@@ -14,36 +14,41 @@
-  }
-
-  public Institution createInstitution(String name, String itDirectorName, String itDirectorEmail, Integer createUser) {
-    Date date = new Date();
-    Integer id = institutionDAO.insertInstitution(name, itDirectorName, itDirectorEmail, createUser, date);
-    return this.findInstitutionById(id);
-  }
+  };
 
-  public Institution createInstitution(String name, String itDirectorName, String itDirectorEmail, Integer createUser, Date date) {
-    Integer id = institutionDAO.insertInstitution(name, itDirectorName, itDirectorEmail, createUser, date);
-    return this.findInstitutionById(id);
+  //NOTE: if NAME column has the UNIQUE attribute in Postgres, can we trust that to throw errors in Java?
+  public Institution createInstitution(Institution institution, Integer userId) {
+      //NOTE: should an error be thrown if id is NOT null? Data may be valid, but id present indicates non-normal request
+      if(institution.getId() != null) {
+        throw new IllegalArgumentException("Cannot pass a value for ID when creating a new Institution");
+      }
+      checkForEmptyName(institution); 
+      Date createTimestamp = new Date();
+      institution.setCreateDate(createTimestamp);
+      institution.setCreateUser(userId);
+      Integer id = institutionDAO.insertInstitution(
+        institution.getName(),
+        institution.getItDirectorName(),
+        institution.getItDirectorEmail(),
+        institution.getCreateUser(),
+        institution.getCreateDate()
+      );
+      return institutionDAO.findInstitutionById(id);
   }
 
-  public void updateInstitutionById(Integer id,
-                                    String institutionName,
-                                    String itDirectorName,
-                                    String itDirectorEmail,
-                                    Integer updateUser) {
+  public Institution updateInstitutionById(Institution institutionPayload, Integer id, Integer userId) {
+    institutionPayload.setId(id);
+    institutionPayload.setUpdateDate(new Date());
+    institutionPayload.setUpdateUser(userId);
+    checkForEmptyName(institutionPayload);
     institutionDAO.updateInstitutionById(
-      id, institutionName, itDirectorName, itDirectorEmail, updateUser, new Date());
-  }
-
-  public void updateInstitutionById(Integer id,
-                                    String institutionName,
-                                    String itDirectorName,
-                                    String itDirectorEmail,
-                                    Integer updateUser,
-                                    Date date) {
-    institutionDAO.updateInstitutionById(
-      id, institutionName, itDirectorName, itDirectorEmail, updateUser, date);
+      institutionPayload.getId(), 
+      institutionPayload.getName(), 
+      institutionPayload.getItDirectorName(), 
+      institutionPayload.getItDirectorEmail(), 
+      institutionPayload.getUpdateUser(),
+      institutionPayload.getUpdateDate());
```

Comment:
Similar feedback here, since we've changed signature, we should do some validation on required fields.
Also, we should be setting the update date here as a new date in the service class and not getting it from the object.